.github/workflows/checkChangelog.yml

```diff
@@ -0,0 +1,19 @@
+name: Check if changelog was edited
+
+on:
+  pull_request:
+    branches:
+      - "*"
+jobs:
+  createReleasePR:
+    name: checkChangelog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+      # What you should use
+      - name: Add simsala
+        run: npm install simsala
+      - name: Check if changlog was updated
+        run: node node_modules/simsala/src/cli.js check --pending-path ./changes
```

Comment:
Where is it triggering here CircleCI? Or are they two separate things? Getting a bit confused here...

Reply:
this is using GithubActions

Reply:
https://github.com/luniehq/lunie/pull/3859/checks?check_run_id=575492006